### ISSUE TITLE
grounding_dino_demo.py performs wrong when set device to other GPUs.

### ISSUE THREAD

**Issue by tingxueronghua** [IDEA-Research/Grounded-Segment-Anything#475]
As described in the title, if setting the device to "cuda:2" there are no results by dino.

Still not sure where the bug occurs.

**Comment by rentainhe** [comment]
Yes, this is something wrong with the Deformable Attention operator, you can try `CUDA_VISIBLE_DEVICES=2` instead of `cuda:2`
